### DISCUSSION TITLE
What is the amount of air the average set of lungs can hold?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#454] [Galaxy Trivia] [answered]
**Bonus:** What does a liter of air weigh?

**Comment by peterbaccaro** [comment]
On average the maximum amount of air your lungs can hold—your total lung capacity—is about **6 litres** but it varies with a person's size, age, gender and respiratory health.
A litre of air weights **1.293 grams** at normal temperature and pressure.

**Comment by BrianMasiga** [comment, marked as answer]
The average set of lungs can hold about 6 liters of air.

A liter of air weighs approximately 1.225 grams at standard temperature and pressure (STP). However, the weight of air can vary depending on the temperature, pressure, and humidity of the air.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/BrianMasiga?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Reply by BrianMasiga** [reply]
Thanks again!

**Comment by thuongtruong1009** [comment]
The amount of air that the average set of lungs can hold, also known as lung capacity, varies based on factors such as age, gender, height, weight, and overall health. However, on average, a healthy adult's lung capacity is around 4-6 liters.

There are several different measurements used to describe lung capacity, including:

Tidal volume: the amount of air that is inhaled and exhaled during normal breathing, which is around 500 milliliters (0.5 liters) for an adult at rest.
Inspiratory reserve volume: the maximum amount of air that can be inhaled after taking a normal breath, which is around 2-3 liters.
Expiratory reserve volume: the maximum amount of air that can be exhaled after breathing out normally, which is around 1-1.5 liters.
Residual volume: the amount of air that remains in the lungs after exhaling completely, which is around 1-1.5 liters.
It's important to note that lung capacity can be affected by a variety of factors, including smoking, lung disease, and other medical conditions, as well as environmental factors such as air pollution.
